content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -16,35 +16,46 @@ apply changes to a cluster.
 
 You can analyze your current live Kubernetes cluster by running:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=analyze_all_namespaces >}}
 $ istioctl analyze --all-namespaces
 {{< /text >}}
 
 And that’s it! It’ll give you any recommendations that apply.
 
 For example, if you forgot to enable Istio injection (a very common issue), you would get the following warning:
 
-{{< text plain >}}
-Warn [IST0102](Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
+{{< text syntax=plain snip_id=analyze_all_namespace_sample_response >}}
+...
+Warn [IST0102] (Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
```

Comment:
Using elision here has no advantage, because the text just says you'll see this message. Elision makes sense when you have a text block that shows the command and expected output, but you want to elide some of it.
```suggestion
Warn [IST0102] (Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
```

Reply:
But the rule output of `analyze all-namespaces` **MAY** include other messages, such as

```
Warn [IST0102] (Namespace ibm-system) The namespace is not enabled for Istio injection. Run 'kubectl label namespace ibm-system istio-injection=enabled' to enable it, or 'kubectl label namespace ibm-system istio-injection=disabled' to explicitly mark it as not needing injection
```

There is no way to predict what `--all-namespaces` will show for a given user.

Reply:
That true, but doesn't matter. The text in the document isn't saying anything about whether or not there will be other output from the command, it's just saying that it there will be a warning like this if injection isn't enabled. For the test program, It's a perfect match for the `_verify_contains` function, which is just checking that the specified message is in the output. The elision is confusing, because the doc at this point doesn't (and shouldn't) mention anything about other possible errors that the command may find. To be perfectly safe for testing in the general case, you would actually need to add `...` both before and after the message of interest, but that's just unnecessary noise in the doc.